hpx/runtime/agas/addressing_service.hpp

```diff
@@ -12,6 +14,7 @@
 #include <hpx/config.hpp>
 
 #include <hpx/exception.hpp>
+#include <hpx/hpx_fwd.hpp>
```

Comment:
Could this be avoided? I would rather prefer for new code to remove hpx_fwd.hpp.